api/src/routes/HttpRoutes.js

```diff
@@ -265,7 +270,20 @@ restify.serve(router, Dashboard);
 restify.serve(router, LRS);
 restify.serve(router, Statement, {
   preCreate: (req, res) => res.sendStatus(405),
-  preDelete: (req, res) => res.sendStatus(405),
+  preDelete: (req, res, next) => {
+    const authInfo = getAuthFromRequest(req);
+    if (!get(process.env, 'ENABLE_SINGLE_STATEMENT_DELETION', true)) {
```

Comment:
If this was set to `false` in the ".env" file this value would be truthy and hence incorrectly allow the deletion of a statement. I suggest using the boolean package to fix this issue since it will parse a "false" string to a falsy value.